cowprotocol/composable-cow
# Add sentry

# Summary

Tenderly Actions logs can't be trusted 100%, they currently have an issue which make us have some blind spots on what is happening. 

for this reason i integrated the WatchTower with tenderly, so we can debug better cases we encounter in production:

<img width="1797" alt="image" src="https://github.com/cowprotocol/cowswap/assets/2352112/5b016788-466d-4a8c-abb2-37b7a6a7ab7f">


Sentry project: https://cowprotocol.sentry.io/projects/watch-tower-tenderly/?project=4505714328928256



# To Test
- Login in Sentry
- Observe the test error i manually created
